src/modules/20-rbac/components/ApiKeyList/views/ApiKeyCard.tsx

```diff
@@ -194,6 +196,15 @@ const ApiKeyCard: React.FC<ApiKeyCardProps> = ({
           {`${getString('common.lastModifiedTime')} `}
           <ReactTimeago date={lastModifiedAt} minPeriod={60} />
         </Text>
+        {defaultTimeToExpireToken && (
+          <Text>
+            {
+              // eslint-disable-next-line strings-restrict-modules
+              getString('platform.authSettings.billingInfo.expires')
```

Comment:
```suggestion
              getString('rbac.token.form.expires')
```
I see you have already updated the strings. 
This will help get rid of `eslint-disable`